### ISSUE TITLE
Broken Link in Readme

### ISSUE THREAD

**Issue by Lakshay-Pruthi** [creativecommons/vocabulary-components#26]
## Description
Vocabulary Storybook link is broken

## Reproduction

1. Go to Readme
![image](https://user-images.githubusercontent.com/91469717/223751170-748849bb-1121-4919-ba8d-360d81f5e3c0.png)

2. Click on this link
3. See error.

## Screenshots
![image](https://user-images.githubusercontent.com/91469717/223751560-80bf7207-c100-4530-aae9-a366c6b4265a.png)

## Environment

 - Browser: Brave

## Resolution

- [x] I would be interested in resolving this bug.

**Comment by vedprakashnautiyal** [comment]
I think I got what your wanted to say, but I think the link itself is not broken since you have to select a particular  component to view more details about that component

**Comment by vedprakashnautiyal** [comment]
I hope you get what I wanted to tell

**Comment by vedprakashnautiyal** [comment]
Instead my suggestion would be to change the link to 
https://cc-vocabulary.netlify.app/?path=/docs/hello-introduction--page

which is the link to the introduction page which tells you to select a component to view more details
<img width="1680" alt="Screenshot 2023-03-08 at 11 55 12 PM" src="https://user-images.githubusercontent.com/71223796/223801860-c44fc3e7-80ac-4c35-820b-77c3a27d4392.png">

**Comment by Lakshay-Pruthi** [comment]
Yeah! Exactly that's what I trying to say and do as well. I think, Maybe there used to be a page "vocabulary-usage--page" which doesn't exist anymore.
![image](https://user-images.githubusercontent.com/91469717/223917775-8be7e4c4-a9e6-47da-8888-7d53f68eaca3.png)

**Comment by possumbilities** [comment]
The storybook site has had a lot of shifting structure and content over time. I agree it could do with a nudge to make this load better. 

Feel free to submit a PR correcting this.

**Comment by vedprakashnautiyal** [comment]
Great ✨
